specification/edgeorderpartner/resource-manager/Microsoft.EdgeOrderPartner/preview/2020-12-01-preview/examples/ManageInventoryMetadata.json

```diff
@@ -0,0 +1,32 @@
+{
+  "parameters": {
+    "subscriptionId": "fa68082f-8ff7-4a25-95c7-ce9da541242f",
+    "location": "westus",
+    "serialNumber": "SerialNumber1",
+    "api-version": "2020-12-01-preview",
+    "User-Agent": [
+      "vscode-restclient"
+    ],
+    "Referer": [
+      "https://localhost/subscriptions/fa68082f-8ff7-4a25-95c7-ce9da541242f/providers/Microsoft.EdgeOrderPartner/locations/westus/inventories/SerialNumber1/manageInventoryMetadata?api-version=2020-12-01-preview"
+    ],
+    "Content-Type": [
+      "application/json; charset=utf-8"
+    ],
+    "Content-Length": [
+      "181"
+    ],
+    "manageInventoryMetadataRequest": {
+      "familyIdentifier": "FamilyIdentifier",
```

Comment:
Examples should contain realistic values please

Reply:
This is fixed.